app/dashboard/utils/route.py

```diff
@@ -225,7 +225,9 @@ def init():
     )
     add_rule(
         "/boot/id/<regex(\"[A-Za-z0-9]{24}\"):uid>/",
-        view_func=vboot.BootIdView.as_view("boot-id"), methods=["GET"])
```

Comment:
Was this change really necessary?

Reply:
Not exactly, but it was not following the code style..

Reply:
OK I think it's better to not include it in this PR as it's not related to removing the deprecated boot bisection support.

Reply:
updated